smac/facade/smac_ac_facade.py

```diff
@@ -492,6 +494,15 @@ def __init__(self,
                 'impute_censored_data': False,
                 'impute_state': None,
             })
+
+        if isinstance(intensifier, SuccessiveHalving) or isinstance(intensifier, Hyperband) \
```

Comment:
Hey, you need to only check for SuccessiveHalving, Hyperband is a subclass of that.

Reply:
Also, I just realized that you need to check for `issubclass` as intensifier is actually a class, not an object.

Reply:
I should check whether `intensifier_instance` is a subclass of `SuccessiveHalving`